yt/visualization/plot_modifications.py

```diff
@@ -65,6 +66,7 @@ def __init__(self, *args, **kwargs):
         pass
 
     def __call__(self, plot):
+        # plot is a CallbackWrapper instance
```

Comment:
This comment reads like a very much missing type annotation (indeed the type of this argument is far from clear from the name). I think there are a couple more places where you could use such a type hint in this module but I won't add individual suggestions everywhere, your IDE is probably a much more confortable place to get them all.
```suggestion
    def __call__(self, plot: CallbackWrapper):
```